assets/capability/capabilities/alarm_problem.json

```diff
@@ -0,0 +1,51 @@
+{
+  "type": "boolean",
+  "title": {
+    "en": "Problem alarm",
+    "nl": "Probleemalarm"
+  },
+  "desc": {
+    "en": "True when a problem is detected.",
+    "nl": "Gaat af wanneer een probleem wordt gedetecteerd."
+  },
+  "insights": true,
+  "insightsTitleTrue": {
+    "en": "Problem alarm turned on",
+    "nl": "Probleemalarm ging af"
+  },
+  "insightsTitleFalse": {
+    "en": "Problem alarm turned off",
+    "nl": "Probleemalarm ging uit"
+  },
+  "getable": true,
+  "setable": false,
+  "uiComponent": "sensor",
+  "$flow": {
+    "triggers": [
+      {
+        "id": "alarm_problem_true",
+        "highlight": true,
+        "title": {
+          "en": "The problem alarm turned on",
+          "nl": "Het probleemalarm gaat aan"
+        }
+      },
+      {
+        "id": "alarm_problem_false",
+        "title": {
+          "en": "The problem alarm turned off",
+          "nl": "Het probleemalarm gaat uit"
+        }
+      }
+    ],
+    "conditions": [
+      {
+        "id": "alarm_problem",
+        "title": {
+          "en": "The problem alarm is !{{on|off}}",
+          "nl": "Het probleemalarm is !{{aan|uit}}"
```

Comment:
Dit is een generiek issue: Ik zou het mooier vinden als we Flow cards hebben als `Er is een probleem` of `Er is geen probleem`